src/main/kotlin/no/nav/familie/ef/sak/mapper/AdresseMapper.kt

```diff
@@ -30,8 +31,10 @@ class AdresseMapper(private val kodeverkService: KodeverkService) {
                           gyldigTilOgMed = adresse.gyldigTilOgMed)
     }
 
-    fun tilAdresse(adresse: Bostedsadresse): AdresseDto {
+    fun tilAdresse(adresse: Bostedsadresse, søkersBostedsadresse: Boolean = false): AdresseDto {
```

Comment:
Forslag på alternativ? 
Grunnen til denne er fordi vi ønsker å returnere bostedsadresseDto for søkeren, men ikke for andre, eks barn eller barns foreldrer

Reply:
Usikker på om jeg forstår hvorfor? For å gjøre søket? Kan vi isåfall gjøre søket basert på personIdent istedenfor?

Reply:
Ja, vi søker personer som bor på samme adresse som søker, og da bruker vi felter som f.eks adressenavn, bruksenhetsnummer osv i søket. 

Hvis jeg skjønner riktig, så informasjon om søker blir ikke lagret noe sted (eller?), og adresse som sendes til front-end er allerede på visningsformat, så derfor i stedet for å hente adresse på nytt fra pdl, vi sender bostedsadresseDto med til front-end.

Om vi skal bruke personIdent, da må vi gjøre søke til pdl 2 ganger, 1 for å finne adresse til søker og 1 for å søke personer som bor på samme adresse....

Reply:
Vi lagrer ikke informasjonen noe sted ennå, men målet er att vi skal det. 
Så tror vi kan ha en mellom-versjon hvor henter dataen 2 ganger, men att den byttes ut mot å hente data fra databasen og sen kalle PDL

Reply:
Vi har endret til att man henter personer/adresser basert på behandlingId